docs/content/new-tutorial/intro-to-assets.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: "Introduction to the Dagster Tutorial and Assets | Dagster Docs"
+description: "Learn about the objectives for the Dagster tutorial and about assets"
+noindex: True
+---
+
+# Introduction to Dagster
+
+Dagster is an orchestrator that's designed for developing and maintaining data assets, such as tables, data sets, machine learning models, and reports.
+
+In this tutorial, you will analyze activity on a popular news aggregation website. Similar to your data pipelines, you will fetch data from the website, clean it up, and build a report that summarizes some findings. We'll then tell Dagster to occassionally update the data and the report, which Dagster calls _assets_.
+
+---
+
+## Introduction to Software-defined assets
+
+Before building a Dagster project, you'll first learn about Dagster's core concept: the Software-defined asset (SDA).
+
+An asset is an object in persistent storage that captures some understanding of the world. Assets can be any type of object, such as:
+
+- A database table or view
+- A file, such as in your local machine or blob storage like Amazon S3
+- A machine learning model
+
+A Software-defined asset is Dagster's concept that couples an asset to the function and upstream assets that are used to produce its contents.
+
+If you have an existing data pipeline, you likely already have assets.
+
+### Goals
+
+By the end of this section, you will:
+
+- Know what assets are
+- Know the power of Software-defined assets
+- Know when to use task-based orchestration with assets
+- Start to think about what parts of your current pipeline are assets
```

Comment:
:think: I keep coming back to this section. I think it's a bit oddly placed, as you've already started describing what assets are in the previous section.

What d'you think about shuffling some things around in this **Introduction to SDAs** section in general? Perhaps something like:

```
## Introduction to Software-defined assets

[Copy up to "If you have an existing data pipeline..."]

With this info in mind, we'll now briefly cover:

- How powerful Software-defined assets can be
- When to use task-based orchestration with assets
- How to think about the parts of your current pipeline that are assets
```

Reply:
You're right in that it's clunky, but I think that's a side effect of putting the tutorial intro and the asset intro on the same page.

Personally, I liked having:
- being as explicit as possible about the checklist of goals, and ya can't get more explicit than a heading that flat out says "GOALS"
- the CTA (Start thinking about assets) in it

I think we can preserve the CTA with your recc, but 1 falls to the wayside with being more implicit. Thoughts on a way to support both?

Reply:
@erinkcochran87 did you have more ideas here?